### PR TITLE
Fixed mario shoes patching when bank 3 is relocated

### PR DESCRIPTION
**Pull request by aglab2**

### CODE REVIEW

File: double_jump/double_jump.asm [hunk 1]
```diff
@@ -4,8 +4,9 @@
 addiu sp, sp, 0xFFE8
 sw ra, 0x14(sp)
 ; Call to our custom function
+lui a2, 0x701C
 jal 0x80406000
-nop
+ori a2, a2, 0x0F00
```
Comment (MathisMARION) [inline double_jump/double_jump.asm:9]: You do not seem to be using this argument.

File: double_jump/double_jump.asm [hunk 2]
```diff
@@ -157,3 +142,13 @@ lw ra, 0x14(sp)
 jr ra
 addiu sp, sp, 0x18
 ; --------------------------
+
+.orga 0x0120614C-0x30
```
Comment (MathisMARION) [inline double_jump/double_jump.asm:146]: Is there a reason for not using a label?

Comment (MathisMARION) [reply double_jump/double_jump.asm:146]: I suppose we'd need a `.headersize 0x80406000 - 0x01206000` directive, is that correct?

File: double_jump/double_jump.asm [hunk 3]
```diff
@@ -157,3 +142,13 @@ lw ra, 0x14(sp)
...
+li t0, 0x80000000
+lw t5, 0x8033b410
```
Comment (MathisMARION) [inline double_jump/double_jump.asm:148]: So, what is this address? I am not too familiar with memory banks.

Comment (MathisMARION) [reply double_jump/double_jump.asm:148]: Ok, I figured out that this is `&sSegmentTable[4]`. I just do not understand yet how data is distributed into various banks. I will assume that Mario is always loaded into bank 4 then, but why does the title mention bank 3 then 🤔

Comment (aglab2) [reply double_jump/double_jump.asm:148]: Indeed, mario is always loaded to bank 4 at fixed segmented address 0x04..... It is technically possible to move mario to a completely different bank but if someone were to do it, they are knowledgeable enough to patch shoes offset as well.

**Pull request closed without merging**